doc/tech/usecases.md

```diff
@@ -135,7 +135,7 @@ CNTT believes that Edge computing is unique in terms of infrastructure requireme
 
   - **Architecture**
 
-      <img width="1089" alt="Screenshot 2020-04-16 at 6 22 58 PM" src="https://user-images.githubusercontent.com/25845305/79684936-635ae400-8235-11ea-9c97-4774aba24d17.png">
+      <img width="1066" alt="Screenshot 2020-05-17 at 11 34 47 PM" src="https://user-images.githubusercontent.com/25845305/82160627-0172b600-9897-11ea-99ec-7339deb19fce.png">
```

Comment:
This image needs to be moved to the appropriate figures directory under CNTT

Reply:
> This image needs to be moved to the appropriate figures directory under CNTT

May I know where is this directory? 

> All acronyms need to be defined on first use.
> 
> All figures need to be under CNTT Github scope.

@pgoyal01 I added the definition for the acronyms (eMBB, UHD, & CDN) in the first use cases.
Could you please advice where is the exact folder/directory to be used for the figures?